src/engines/html5/html5.js

```diff
@@ -936,3 +937,5 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     return null;
   }
 }
+
+registerEngine(Html5);
```

Comment:
Check if html5 is supported before register it
`if (Html5.isSupported())`

Reply:
Today our capabilities framework already include isSupported if we wish to change it  - lets add a task for it :-)

Reply:
I think what @dan-ziv means is to not add HTML5 as an option if it's not supported.
Today, as we didn't add any support to platform that didn't support HTML5/MSE playback we didn't care about this.